src/installer/pkg/projects/netcoreapp/pkg/legacy/Microsoft.NETCore.App.Internal.pkgproj

```diff
@@ -1,28 +0,0 @@
-<Project>
-
-  <PropertyGroup>
-    <!-- Make this Internal package's runtime.json point to the non-Internal runtime packages. -->
-    <RuntimeProjectFile>$(MSBuildThisFileDirectory)Microsoft.NETCore.App.pkgproj</RuntimeProjectFile>
```

Comment:
There’s just the ones that use it for dependency flow. We’ll update those to use either the VS insertion package or the extensions transport package.

Reply:
Here's a GitHub search for the packages that depend on the Microsoft.NETCore.App.Internal package: https://github.com/search?q=Microsoft.NETCore.App.Internal+user%3Adotnet+path%3Aeng+filename%3AVersion.Details.xml&type=Code&ref=advsearch&l=&l=

WPF can move to using Microsoft.NETCore.App.Runtime.win-x64. Installer needs to move to another non-shipping package.

Source-Build is a false positive.

EF6 is pinned at 3.1.0, so I don't think we need to worry about it right now.

Reply:
I've opened PRs in [WPF](https://github.com/dotnet/wpf/pull/3791) and [Installer](https://github.com/dotnet/installer/pull/9059) to remove the usage.

Reply:
diagnostics uses the non-internal version for non-stable version. Is that one also going?

Reply:
not yet (TM)

Reply:
Yes the Microsoft.NETCore.App package is also being removed. I'll send a PR to diagnostics to update it.

Reply:
Diagnostics PR open at https://github.com/dotnet/diagnostics/pull/1712

Reply:
Oh I thought the legacy shared framework package isn't removed yet?

Reply:
No the legacy package is being removed in this PR.

Reply:
Are you sure removing it is safe? At some point i thought the empty package was needed (regardless of the internal package)

Reply:
No it was only needed for the 2.x way of referencing the runtime and the way of contstructing the shared framework in master. We haven't shipped a Microsoft.NETCore.App package to nuget.org since 2.x.

Reply:
Ok, sounds good then.